frontend/src/components/cars-categories/cars-categories.tsx

```diff
@@ -2,21 +2,40 @@ import React, { FC, useState } from 'react';
 import { Link } from 'react-router-dom';
 
 import { mockBrands } from '@components/cars-categories/mock-brands';
+import { Mousewheel, Navigation, SwiperOptions } from 'swiper';
+import { Swiper, SwiperSlide } from 'swiper/react';
 
 import { Brand } from '../../types/brand.type';
+import 'swiper/css';
+import 'swiper/css/navigation';
+import 'swiper/css/mousewheel';
 import styles from './styles.module.scss';
 
 const CarsCategories: FC = () => {
   const [brands] = useState<Brand[]>(mockBrands);
 
+  const swiperParams: SwiperOptions = {
+    slidesPerView: 8,
+    loop: true,
+    mousewheel: true,
+    navigation: true,
+    modules: [Navigation, Mousewheel],
+  };
+
   return (
-    <div className={styles.container}>
+    <Swiper className={styles.container} {...swiperParams}>
```

Comment:
The static object like this one should be declared either **_outside_** of the component or **_in the `useMemo`_**. It's because on each render this object will be recreated and will inevitably lead to the `Swiper`'s rerender